src/Microsoft.Health.Operations.Functions/Management/PurgeOrchestrationInstanceHistory.cs

```diff
@@ -0,0 +1,74 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Core;
+using Microsoft.Health.Functions.Extensions;
+
+namespace Microsoft.Health.Operations.Functions.Management;
+
+/// <summary>
+/// Represents a periodic process that purges stale orchestration instance metadata from storage.
+/// </summary>
+public sealed class PurgeOrchestrationInstanceHistory
+{
+    private readonly PurgeHistoryOptions _options;
+
+    private const string PurgeFrequencyVariable = $"%{AzureFunctionsJobHost.RootSectionName}:{PurgeHistoryOptions.SectionName}:{nameof(PurgeHistoryOptions.Frequency)}%";
```

Comment:
Unfortunately this means that the configuration path is hard-coded, so all uses must use this path in their configs.